prime-dotnet-webapi/Models/LicenseClassClause.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Prime.Models.AccessAgreement;
+
+namespace Prime.Models
+{
+    [Table("LicenseClassClause")]
+    public class LicenseClassClause : BaseAuditable, IAccessClause
+    {
+        public LicenseClassClause()
+        {
+            // Create lists so they don't have be instantiated when items need to be added
+            TermsOfAccessLicenseClassClauses = new List<TermsOfAccessLicenseClassClause>();
+        }
+
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public string Clause { get; set; }
+
+        [Required]
+        public DateTime EffectiveDate { get; set; }
+
+        public List<TermsOfAccessLicenseClassClause> TermsOfAccessLicenseClassClauses { get; set; }
```

Comment:
I think you mentioned having problems with circular references in this PR? This is why. When you JSONify your TOA, it JSONifys all the LicenceClassClauses through the xref table/navigation property, each of which have a ref back to the Xref table, which is circular. you should [JsonIgnore] the navigation property here.

Reply:
Nice thanks James!  That was super frustrating